### ISSUE TITLE
Problems in "require ();"

### ISSUE THREAD

**Issue by joaomarcusjesus** [Wizcorp/phonegap-facebook-plugin#1331]
# Hi guys!

I am having a problem
 in an application using intel xdk to use plguin. And the following problem happens

"Uncaught ReferenceError: require is not defined"
var exec = require("cordova.exec");

Has this happened to anyone?

# Thanks :)

**Comment by dnoliver** [comment]
Hello @joaomarcusjesus, 

can you post some code to see what is happening? I think that problem is not related to this plugin, but it is a problem with the application code itself

**Comment by joaomarcusjesus** [comment]
# Thanks @dnoliver 
I've solved the problem. Yes, it was not the plugin. But as I researched that "codorva.js" there is no browser but yes in the application.

**Comment by dondragon2** [comment]
how did you solve this? I am getting the same on android device.

**Comment by joaomarcusjesus** [comment]
@dondragon2  
My problem was that I was trying to test FacebookConnectPlugin in a browsers. But my application used cordova.js and it does not work in browsers. So I managed the apk and tested it on my cell phone

This solves your problem??

**Cya**

**Comment by dondragon2** [comment]
No. It works in the browser but is failing on the device on that line.

**Comment by joaomarcusjesus** [comment]
Okay, I'll go find it and give it a go.

**Cya**

**Comment by renanbastos93** [comment]
Hello guys,

I have problem equals of you, please help.

The device, error "require", because in browser is success, using IONIC

thanks
